AutoRest/Generators/CSharp/Azure.CSharp.Tests/Expected/SwaggerBat/Lro.Cs/ILROSADsOperations.cs

```diff
@@ -45,7 +45,8 @@ public partial interface ILROSADsOperations
         Task<AzureOperationResponse<Product>> GetNonRetry400WithOperationResponseAsync(CancellationToken cancellationToken = default(CancellationToken));
         /// <summary>
         /// Long running put request, service returns a Product with
-        /// 'ProvisioningState' = 'Creating' and 201 response code
+        /// &apos;ProvisioningState&apos; = &apos;Creating&apos; and 201
```

Comment:
Do we really want to escape the qotes in documentation. It makes it hard to read.

Escaping formwardslash, backwardslash and star_forwardslash  would be fine